src/ZoneServer/World/Items/Item.cs

```diff
@@ -112,6 +112,25 @@ public Item(int itemId, int amount = 1)
 			this.Amount = amount;
 		}
 
+		/// <summary>
+		/// Copy constructor
+		/// </summary>
+		/// <param name="other"></param>
+		public Item (Item other)
+		{
+			this.Id = other.Id;
+			this.LoadData();
+			this.Price = other.Price;
+			this.IsLocked = other.IsLocked;
+			this.OriginalOwnerHandle = other.OriginalOwnerHandle;
+			this.RePickUpTime = other.RePickUpTime;
+			this.OwnerHandle = other.OwnerHandle;
+			this.LootProtectionEnd = other.LootProtectionEnd;
+			this.Properties = other.Properties;
```

Comment:
You shouldn't just copy the reference, because changes to one item will then affect the copies as well and vice versa. Instead, I added a `CopyFrom` method to `Properties` a while back for this purpose.